peps/pep-0764.rst

```diff
@@ -262,45 +282,28 @@ cases incompatible, especially for runtime introspection::
     # Raises a type error at runtime:
     def fn() -> {'a': int} | int: ...
 
-Open Issues
-===========
-
-Subclassing an inlined typed dictionary
----------------------------------------
-
-Should we allow the following?::
-
-    from typing import TypedDict
-
-    InlinedTD = TypedDict[{'a': int}]
+Extending other typed dictionaries
+----------------------------------
 
-
-    class SubTD(InlinedTD):
-        pass
-
-What about defining an inlined typed dictionay extending another typed
-dictionary?::
+Several syntaxes could be used to have the ability to extend other typed
+dictionaries::
 
     InlinedBase = TypedDict[{'a': int}]
 
     Inlined = TypedDict[InlinedBase, {'b': int}]
+    # or, by providing a slice:
+    Inlined = TypedDict[{'b': int} : (InlinedBase,)]
```

Comment:
Black and ruff would reformat this as `TypedDict[{'b': int}: (InlinedBase,)]`. And even though it's now formatted exactly like in [my proposal](https://discuss.python.org/t/pep-764-inlined-typed-dictionaries/78779/13), it might be worth to consider a black-compatible formatting instead 🤷🏻

Reply:
We can just change this in Black, current behavior of formatters shouldn't unduly influence language evolution.

Reply:
> We can just change this in Black, current behavior of formatters shouldn't unduly influence language evolution.

even better!

Reply:
Note that this is part of the rejected ideas, as per https://discuss.python.org/t/78779/18.